conans/client/build/msbuild.py

```diff
@@ -116,6 +118,9 @@ def get_command(self, project_file, props_file_path=None, targets=None, upgrade_
         if toolset:
             command.append('/p:PlatformToolset="%s"' % toolset)
 
+        if verbosity:
```

Comment:
I think it will not be true if you set `CONAN_MSBUILD_VERBOSITY` to the empty string, for instance, right? (I suppose python evaluates empty strings to false).

Reply:
leaving check as it is not always true